av/video/stream.pyx

```diff
@@ -10,3 +10,12 @@ cdef class VideoStream(Stream):
             self.codec_context.height,
             id(self),
         )
+
+    property framerate:
+        """Explicitly block pass through access to the framerate property."""
+        def __get__(self):
+            raise AttributeError("framerate")
+
+    cdef _set_framerate(self, value):
```

Comment:
I really don't understand this, I've never seen a setter declared like this? Is there some Cython magic at play here?

Reply:
I don't know actually - that's just how it was done with the existing properties.

Reply:
No,  the general pattern is that you have a __set__ at the same level as your __get__, or the more modern syntax is just with @property like regular python ;)

Reply:
LOL yeah, you're right. It was late where I am and I actually copied the syntax from the _set_id workaround code for setattr (which was actually from my own work before :-P). Anyway it's irrelevant now since this section has been removed.